src/index.js

```diff
@@ -70,7 +70,7 @@ function AppWrapper() {
         <Web3Wrapper>
           <App>
             <Route path="/" exact component={Home} />
-            <Route path="/deposit" exact component={StartDeposit} />
+            {/* <Route path="/deposit" exact component={StartDeposit} />
             <Route path="/deposit/new" component={Invoice} />
             <Route path="/deposit/:address/get-address">
```

Comment:
We could disable routes for deposit creation only and don't touch the paths for deposit creation continuation, in case we upgrade the dapp while a deposit is already in creation.

Reply:
067297b